dbbackup/management/commands/dbbackup.py

```diff
@@ -43,16 +45,29 @@ def handle(self, **options):
         self.compress = options.get('compress')
         self.encrypt = options.get('encrypt')
         self.storage = BaseStorage.storage_factory()
-        database_keys = (self.database,) if self.database else dbbackup_settings.DATABASES
-        for database_key in database_keys:
-            database = settings.DATABASES[database_key]
-            self.dbcommands = DBCommands(database)
+        if options.get('mongo'):
+            # Mongo specific handling.
+            # TODO: Add support for multiple mongo db defined in settings.
```

Comment:
It just seemed a bit overkill to use django-mongod-engine when I just wanted to connect to mongo, having a separate settings and module wasn't a pain.

I personnaly think this command should support both mongo in DATABASES or defined somewhere else. Probably create a separate command for nosql with dedicated backup settings is the best option.